compass/ocean/tests/soma/namelist.forward

```diff
@@ -0,0 +1,27 @@
+config_ocean_run_mode = 'forward'
+config_run_duration = '0000_00:01:30'
```

Comment:
`dt` is at least 6 minutes, usually 10 minutes.  This should be updated to be several time steps.